zScanner/Services/Storage/Model/DcumentTypeDatabaseModel.swift

```diff
@@ -12,19 +12,16 @@ import RealmSwift
 class DocumentTypeDatabaseModel: Object {
     @objc dynamic var id = ""
     @objc dynamic var name = ""
-    @objc private dynamic var _mode = ""
     
     convenience init(documentType: DocumentTypeDomainModel) {
         self.init()
         
         self.id = documentType.id
         self.name = documentType.name
-        self.mode = documentType.mode
     }
-
-    var mode: DocumentMode {
-        get { return DocumentMode(rawValue: _mode)! }
-        set { self._mode = newValue.rawValue }
+    
+    override class func primaryKey() -> String {
+        return "id"
```

Comment:
Are you sure about this change? Why did you add primary key?

Reply:
I'm not sure about it but it says that the DocumentTypeDatabaseModel needs to have a primary key and it doesn't work without it.